src/avx512-common-qsort.h

```diff
@@ -671,14 +671,97 @@ static inline int64_t partition_avx512(type_t1 *keys,
 }
 
 template <typename vtype, typename type_t>
-void qsort_(type_t *arr, int64_t left, int64_t right, int64_t maxiters);
+X86_SIMD_SORT_INLINE type_t get_pivot(type_t *arr,
```

Comment:
We don't need a separate declaration. You can move the definition of `get_pivot `here.